warehouse/clickhouse/clickhouse.go

```diff
@@ -196,32 +201,31 @@ func (ch *HandleT) newClickHouseStat(tableName string) *clickHouseStatT {
 }
 
 func Init() {
-	loadConfig()
 	pkgLogger = logger.NewLogger().Child("warehouse").Child("clickhouse")
 }
 
-// Connect connects to warehouse with provided credentials
-func Connect(cred CredentialsT, includeDBInConn bool) (*sql.DB, error) {
+// ConnectToClickhouse connects to clickhouse with provided credentials
+func (ch *Handle) ConnectToClickhouse(cred Credentials, includeDBInConn bool) (*sql.DB, error) {
 	var dbNameParam string
 	if includeDBInConn {
 		dbNameParam = fmt.Sprintf(`database=%s`, cred.DBName)
 	}
 
-	url := fmt.Sprintf("tcp://%s:%s?&username=%s&password=%s&block_size=%s&pool_size=%s&debug=%s&secure=%s&skip_verify=%s&tls_config=%s&%s&read_timeout=%s&write_timeout=%s&compress=%t",
+	url := fmt.Sprintf("tcp://%s:%s?&username=%s&password=%s&block_size=%s&pool_size=%s&debug=%s&secure=%s&skip_verify=%s&tls_config=%s&%s&read_timeout=%s&write_timeout=%s&Compress=%t",
```

Comment:
Is there a reason for renaming `compress` to `Compress`?

Reply:
It happened by mistake. Thanks for pointing that out. reverting this.

Reply:
We could use `url.Values` for constructing the URI, it would be more readable and we won't have any issue with unescaped values.